chapters/resampling.qmd

```diff
@@ -584,6 +586,144 @@ Should we use the bootstrap to compare models? Its small variance is enticing, b
 
 Let's take a look at a few specialized resampling methods. 
 
+
+## Time Series Data {#sec-time-series-resampling}
+
+Time series data [@Hyndmanforecast] are sequential observations ordered over time. The most common example is daily stock prices. These data are a special case due to _autocorrelation_: rows of the data set are not independent. This can occur for many different reasons (e.g. underlying seasonal trends, etc.), but it complicates the resampling process in different ways: 
+
+ - We want to avoid analysis and assessment partitions where older data are in the assessment set (to avoid predicting past data with future data). 
+ - Random sampling might destroy the underlying data structures that can be used to measure specific trends (such as seasonality). 
+
+_Rolling forecasting origin_ splitting [@tashman2000out] is a resampling scheme that mimics the train/test splitting pattern from @sec-basic-splitting where the most recent data are used in the assessment set. We would choose how much data (or what range of time) to use for the analysis and assessment sets, then slide this pattern over the whole data set. @fig-rolling-scheme illustrates this process where each block could represent some unit of time. For example, we might build the model on six months of data and then predict with one month. The next resample iteration could bump that forward by a month or another period (so that the assessment sets can be distinct or overlapping).
+
+```{r}
+#| label: fig-rolling-scheme
+#| echo: false
+#| out-width: '55%'
+#| fig-cap: "A schematic of rolling origin resampling for time series data. Each box corresponds to a unit of time (e.g., day, month, year, etc.)."
+knitr::include_graphics("../premade/rolling.svg")
+```
+
+We also have the choice to cumulatively expand the assessment data from the start of the training set to the end of the current slice.
```

Comment:
```suggestion
We also have the choice to cumulatively expand the assessment data from the start of the training set to the end of the current slice.  For example, in Resample 2 of @fig-rolling-scheme, the model would be fit using samples 1-9; in Resample 3, the model would be fit using samples 1-10; and so on.  This approach may be useful when the number of periods of time are smaller and we would like to include as much of the historical data as possible in each resample.
```

Reply:
In the first sentence, should it be:  ...cumulatively expand the analysis data...  ?

Reply:
Yes sir, you are correct 😄